automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -202,7 +202,7 @@ class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
       val petEmail = getAndVerifyPet(runtimeFixture.runtime.googleProject)
       // cluster should have been created with the pet service account
 
-      runtimeFixture.runtime.serviceAccountInfo.clusterServiceAccount shouldBe Some(petEmail)
+      runtimeFixture.runtime.serviceAccountInfo shouldBe petEmail
 
       withWebDriver { implicit driver =>
         withNewNotebook(runtimeFixture.runtime, Python3) { notebookPage =>
```

Comment:
I think you can also remove `NotebookTestUtils.verifyNotebookCredentials()` in automation tests. That method is for verifying the notebook service account.